R/extend_ae_specific.R

```diff
@@ -333,6 +334,18 @@ extend_ae_specific_events <- function(outdata) {
 
   outdata$events <- avg
   outdata$events_se <- se
+  outdata$extend_call <- c(outdata$extend_call, match.call())
+
+  outdata
+}
+
+#' Add Subgroup analysis in AE specific analysis
+#'
+#' @param subgroup a character string for subgroup variable name
+#'
+#' @export
```

Comment:
- Should "subgroup" in the title be in lower case?
- `@params` description should use sentence case with a full stop.
- Exported functions should have `@return` and `@examples` sections.